src/test/java/org/haffson/adventofcode/days/day01/Day01Test.java

```diff
@@ -20,28 +22,108 @@
     @Value("${day1.file}")
     private String filePath;
 
+    private Day01 day01;
+
+    @Before
+    public void before() {
+        this.day01 = new Day01();
+    }
+
     public void testGetDay() {
-        Day01 day01 = new Day01(fileReaders, filePath);
         int expectedResult = 1;
-        int actualResult = day01.getDay();
-        Assert.assertEquals(expectedResult, actualResult);
+
+        int actualResult = this.day01.getDay();
+
+        assertThat(expectedResult).isEqualTo(actualResult);
+    }
+
+    @Test
+    public void testMultiplyListEntries() {
+        int[] values = new int[]{979, 366, 675};
+        int expectedResult = 241861950;
+
+        int actualResult = this.day01.multiplyListEntries(values);
+
+        assertThat(actualResult).isEqualTo(expectedResult);
+    }
+
+    @Test
+    public void testMultiplyListEntriesWithEmptyList() {
+        int[] values = new int[]{};
+        int expectedResult = 0;
+
+        int actualResult = this.day01.multiplyListEntries(values);
```

Comment:
If you are using your methods like this, and providing the values from the outside (rather than using the class variable `numbers`, see my comment above), you might as well make these methods static and thus not dependent on an instance of the class Day01.

Also note that normally the `this` keyword is not used like this. It is only needed if you have the same variable coming e.g. as a method parameter and want to differentiate if you mean `this` class or (without keyword) the locally scoped variable.